src/System.ValueTuple/src/System/Numerics/Hashing/Hash.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Numerics.Hashing
+{
+    internal static class Hash
```

Comment:
@stephentoub Ok; I re-added `ValueTuple.CombineHashCodes` and had it forward to the helper class. The diff is now only ~60 LOC.